src/layout.sh

```diff
@@ -156,7 +156,24 @@ start_listener() {
 
         [[ "$source" != "$dest" ]] && __recalculate_layout;
       else
-        __recalculate_layout;
+        desk_file="/tmp/bsp-layout_desktop.txt"
+
+        if [ ! -f "$desk_file" ]; then
```

Comment:
This whole thing is pointless. I can't seem to wrap my head around what purpose it serves

Reply:
that is precisely the point of my previous messages on that PR... :thinking: 
and without @darkelectron to explain what he had in mind, i do not get the idea of that snippet

do we remove that?
in that case, what is the point of that PR apart from adding `node_flag` and `node_state`?

Reply:
Yeah. Those 2 events are important. The rest is safe to remove.

I think the purpose was to prevent it from recalculating layout in a certain case but I think a redundant calculation is fine for now.

Reply:
that's fine with me :ok_hand: 

i'll revert the commit and push that again, then you might be good to approve as there is only the `node_flag` and `node_state` that get added through that PR :wink: